onnxruntime/test/optimizer/qdq_test_utils.h

```diff
@@ -71,7 +84,8 @@ GetQDQTestCaseFn BuildQDQConvTransposeTestCase(const std::vector<int64_t>& input
                                                 dq_w_output);
 
     auto* dq_bias_output = builder.MakeIntermediate();
-    auto* bias = builder.MakeInitializer<BiasType>({weights_shape[0]}, static_cast<BiasType>(0), static_cast<BiasType>(127));
+    auto* bias = builder.MakeInitializer<BiasType>({weights_shape[0]}, static_cast<BiasType>(0),
+                                                   static_cast<BiasType>(127));
```

Comment:
Note: The Github linter threw an error because too many lines exceeded the limit of 120 characters. It seems the linter can only report so many line length violations. I had to fix them in this file to allow the linter to finish.